status-im/status-react
# Wallet - Add support for Sepolia testnet and Rarible collectible provider

fixes #18188
fixes #18510 

### Summary

This PR adds support for the Sepolia test network and Rarible collectible/collection provider.

#### :warning: Notes

- The `network` and `network info` in the advanced settings will be removed soon (shortly in a separate PR) as it's from the old wallet. Please don't confuse it with new wallet settings (`Enable Testnet mode`, and `Enable Sepolia as Test network`).

### Platforms

- Android
- iOS

#### Areas impacted

##### Functional
- wallet

### Steps to test

- Open Status
- Navigate to `Profile > Advanced Settings`
- Enable `Testnet mode`
- Enable `Sepolia as Test network`
- Verify the token balances displayed including Sepolia network balances
- Verify collectables are displayed from the Sepolia testnet

#### Testing Notes:

 - Sepolia faucet https://sepolia-faucet.pk910.de/
 - Collectibles https://www.testnetmint.com/
 - Rarbile collectibles currently support ETH (Mainnet and Goerli), and ARB (Mainnet and Sepolia)

status: ready
